src/modules/logicModule/IBountyManager.sol

```diff
@@ -0,0 +1,266 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import {IPaymentClient} from "src/modules/base/mixins/IPaymentClient.sol";
+
+interface IBountyManager is IPaymentClient {
+    //--------------------------------------------------------------------------
+    // Enums
+
+    enum Roles {
+        BountyAdmin,
+        ClaimAdmin,
+        VerifyAdmin
+    }
+
+    //--------------------------------------------------------------------------
+    // Types
+
+    struct Bounty {
+        /// @dev Minimum amount of tokens that can be paid out upon fulfillment of the bounty
+        uint minimumPayoutAmount;
+        /// @dev Maximum amount of tokens that can be paid out upon fulfillment of the bounty
+        uint maximumPayoutAmount;
+        /// @dev Arbitrary data to store Bounty details if necessary.
+        ///      CAN be empty.
+        bytes details;
+        /// @dev Id that claimed the bounty
+        ///      A Bounty is claimed if a Claim got acknowledged by a Verifier
+        uint claimedBy;
+    }
+
+    struct Claim {
```

Comment:
Perhaps, it will be a good idea to keep the `Contributor` struct above struct `Claim`. It simply reads better.